db/raw/master-head.sql

```diff
@@ -132,12 +148,13 @@ CREATE TABLE attendee (
     payment_method integer,
     badge_printed_name character varying DEFAULT ''::character varying NOT NULL,
     staffing boolean DEFAULT false NOT NULL,
```

Comment:
I removed the ``fire_safety_cert`` because we're not even asking for it as part of the Volunteer checklist anymore, and even if we were, that belongs in a MAGFest repo and not in the core repo.